app/org/sagebionetworks/bridge/validators/ConsentSignatureValidator.java

```diff
@@ -22,11 +30,29 @@ public boolean supports(Class<?> clazz) {
     @Override
     public void validate(Object target, Errors errors) {
         ConsentSignature sig = (ConsentSignature) target;
+        
         if (Strings.isNullOrEmpty(sig.getName())) {
             errors.rejectValue("name", CANNOT_BE_BLANK);
         }
-        if (Strings.isNullOrEmpty(sig.getBirthdate())) {
-            errors.rejectValue("birthdate", CANNOT_BE_BLANK);
+        LocalDate birthdate = parseBirthday(sig.getBirthdate());
+        // There was a value, but it didn't parse
+        if (birthdate == null && !Strings.isNullOrEmpty(sig.getBirthdate())) {
+            errors.rejectValue("birthdate", "is invalid (required format: YYYY-MM-DD)");
+        }
```

Comment:
Would it be cleaner to check that sig.getBirthdate() is specified and *then* try to parse it instead of the other way around?

Reply:
Changed